Superlib.js

```diff
@@ -0,0 +1,633 @@
+{
+	"translatorID": "33553736-4ea9-4b1a-b1e1-d43b2a918156",
+	"label": "Superlib",
+	"creator": "018<lyb018@gmail.com>",
+	"target": "https?://(book|jour)\\.ucdrs\\.superlib\\.net/(search|views/specific)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-09-13 12:28:57"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 018<lyb018@gmail.com>
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+// eslint-disable-next-line
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}function trim(content){return content.replace(/^[\xA0\s]+/gm, '').replace(/[\xA0\s]+$/gm, '').replace(/\n+/g, '\n').replace(/:\n+/g, ': ').replace(/]\n/g, ']').replace(/】\n/g, '】').replace(/\n\/\n/g, '/')}
+
+// https://aurimasv.github.io/z2csl/typeMap.xml#map-book
+// https://aurimasv.github.io/z2csl/typeMap.xml#map-conferencePaper
+// https://aurimasv.github.io/z2csl/typeMap.xml#map-thesis
+// https://aurimasv.github.io/z2csl/typeMap.xml#map-journalArticle
+// https://aurimasv.github.io/z2csl/typeMap.xml#map-patent
+var TYPE_MAP = {
+	图书: 'book',
+	会议论文: 'conferencePaper',
+	学位论文: 'thesis',
+	期刊: 'journalArticle',
+	专利: 'patent'
+};
+
+function doPerson(item, data) {
+	// （日）本田晃一著；朱运程译 translator author
+	if (!data || data.length <= 0) return;
```

Comment:
This check (and the other ones like it) can, I would think, just be simplified to `if (!data)`.